src/Components/Browser.JS/src/Boot.Server.ts

```diff
@@ -110,8 +110,10 @@ async function initializeConnection(options: Required<BlazorOptions>, circuitHan
   (hubProtocol as unknown as { name: string }).name = 'blazorpack';
 
   const connectionBuilder = new signalR.HubConnectionBuilder()
-    .withUrl('_blazor')
-    .withHubProtocol(hubProtocol);
+    .withUrl('_blazor', {
+      ensureMessageSendOrder: true
```

Comment:
Are we making this non-default because of perf considerations? If so, that could probably be mitigated by batching multiple sends per post requests if we start falling behind.

I like making this the default, because it provides more consistency between the WebSocket transport vs. SSE/LP.

Reply:
@BrennanConroy suggested making this opt-in due to perf considerations. If we do want to make this default:

* Is the value "true" in the options by default
* Or is there an inverse option and the user has a gesture to turn it off?  `disableHttpTransportSendQueue: true`?

Reply:
Ideally, we get the perf good enough with this change that we don't need an option to disable it. If that doesn't work, I think opt-out is better than opt-in.

Reply:
I don't remember complaining about perf here.

The main reason for opt-in is that it's a change in behavior, although it's a change in behavior to be more correct.
I'm ok with it being on by default. cc @anurse